Tribute Page/index.html

```diff
@@ -75,8 +75,8 @@ <h1>Stevie Ray Vaughan</h1>
             over 15 million albums in the United States alone. In 2003, David
             Fricke of Rolling Stone ranked him the seventh greatest guitarist of
             all time. Vaughan was posthumously inducted into the Rock and Roll
-            Hall of Fame in 2015, along with Double Trouble bandmates Chris
-            Layton, Tommy Shannon, and Reese Wynans.<br />
+            Hall of Fame in 2015, along with Double Trouble bandmates, the drummer Chris
```

Comment:
The way the musicians from Double Trouble have their function at the band explained is better